wolox-books/src/app/screens/book/components/list/list.component.html

```diff
@@ -1,8 +1,8 @@
 <div class="search">
-  <input type="text" class="input-search" placeholder="Busc&aacute; por t&iacute;tulo del libro...">
+  <input type="text" class="input-search" placeholder="Busc&aacute; por t&iacute;tulo del libro..." [(ngModel)]="search">
 </div>
 
-<div *ngFor="let book of books | async" class="book-container">
+<div *ngFor="let book of books | async | filterBook:search" class="book-container">
   <div class="book-card" *ngFor="let number of [0,1,2,3,4,5]">
     <img src="../../../../../assets/book-cover.png" alt="" class="book-cover">
```

Comment:
close tag

Reply:
raise an angular error as the latest PRs

Reply:
I don't remind that error 😅